contrib/completion/bash/docker

```diff
@@ -4226,6 +4227,17 @@ _docker_manifest_push() {
 	esac
 }
 
+_docker_network_rm() {
```

Comment:
A copy & paste error:
```suggestion
_docker_manifest_rm() {
```

Reply:
I was hoping that this would be fixed before merging the PR.

Reply:
omg I made the change locally but forgot `git add`

https://github.com/docker/cli/pull/2728

Reply:
Arf; my bad; I saw the comment collapsed, and thought it was updated 😞